hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -1783,6 +1783,8 @@ func reconcileHostedControlPlane(hcp *hyperv1.HostedControlPlane, hcluster *hype
 	}
 	if hcluster.Spec.SecretEncryption != nil {
 		hcp.Spec.SecretEncryption = hcluster.Spec.SecretEncryption.DeepCopy()
+	} else {
```

Comment:
can we have a validateAzureKMSConfig analogous to validateAWSKMSConfig?

Reply:
done!